frontend/src/pages/chat/components/chat-sidebar/chat-sidebar.tsx

```diff
@@ -57,6 +60,16 @@ const ChatSidebar: React.FC<Properties> = ({
     void dispatch(chatsActions.getAllChats(filter));
   }, [dispatch, filter]);
 
+  useEffect(() => {
+    if (chatsDataStatus === DataStatus.FULFILLED) {
+      setIsChatsLoaded(true);
+    }
+  }, [chatsDataStatus, setIsChatsLoaded]);
```

Comment:
why do we need a state for this? Can't we simply check like this?
```ts
const isChatsLoaded = chatsDataStatus === DataStatus.FULFILLED
```

Reply:
@HIMDanny We can't. Scroll to the selected card should be only on first chats loading, because then data status will change on every filter request

Reply:
now I see, thanks for clarification

Reply:
btw, we also can ref for this to not cause a rerender after `setIsChatLoaded(true)`

Reply:
@HIMDanny What do you mean by saying use ref?

Reply:
instead of using state for setting this boolean value, we can make something like:
```ts
const isChatLoaded = useRef(false);
...
isChatLoaded.current = true;
```

Reply:
nevermind, now I am not sure whether it will work)